_docs_operate/modules.md

```diff
@@ -87,6 +87,12 @@ For this, the Connector supports the configuration of webhooks which are called
 
 Keep in mind that you need to synchronize the state of the Connector with the Backbone in order to receive events. The `sync` Module automates this, but you can also do this manually by calling the `/api/v2/Account/Sync` route.
 
+### sse (sever sent events) <a href="{% link _docs_operate/configuration.md %}#sse"><i class="fas fa-fw fa-cog"/></a> {#sse}
+
+The sse Module allows the Connector to open an one way connection from the Backbone to the Connector.
```

Comment:
IMO the whole description of the module is way to technical. And the technical details are wrong. The Backbone isn't the actor that opens a connection to the Connector - it is the other way round. As someone who reads this I would directly see this module as unusable as I don't want to expose my connector to the internet.